runtime/literal.go

```diff
@@ -101,13 +101,18 @@ func arrayLiteralValue(elements []ast.Expression, elementType sema.Type) (cadenc
 	return cadence.NewArray(values), nil
 }
 
-func pathLiteralValue(expression ast.Expression, ty sema.Type) (cadence.Value, error) {
+func pathLiteralValue(expression ast.Expression, ty sema.Type) (result cadence.Value, errResult error) {
 	pathExpression, ok := expression.(*ast.PathExpression)
 	if !ok {
 		return nil, LiteralExpressionTypeError
 	}
 
-	pathType, err := sema.CheckPathLiteral(pathExpression)
+	pathType, err := sema.CheckPathLiteral(
+		pathExpression.Domain.Identifier,
+		pathExpression.Identifier.Identifier,
+		ast.NewRangeFromPositioned(pathExpression.Domain),
+		ast.NewRangeFromPositioned(pathExpression.Identifier),
```

Comment:
The ranges are only needed for errors, which is the unlikely case. 

Maybe refactor the range parameters to functions of type `func () ast.Range`